<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_12">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,7">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_showMenu" stretch="1,0,0,0,0,0,0,0,0,0,20">
       <item>
        <spacer name="verticalSpacer_37">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="AllName">
         <property name="font">
          <font>
           <family>幼圆</family>
           <pointsize>22</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(117, 103, 127);</string>
         </property>
         <property name="text">
          <string>智慧教育系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_40">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="2,6">
         <item>
          <widget class="QLabel" name="label_21">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>当前页面:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="now_page">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_39">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="2,6">
         <item>
          <widget class="QLabel" name="frame_num">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>帧率:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="frame_lcdNumber">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_41">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="2,6">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>手势:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="hand_mod">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_42">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="4,6">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string>指令:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="construct">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_38">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_showPic" stretch="0,8,0,2">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;---------画面显示---------&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGraphicsView" name="ShowImage_">
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background: rgb(0, 0, 0)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="backgroundBrush">
          <brush brushstyle="SolidPattern">
           <color alpha="0">
            <red>29</red>
            <green>29</green>
            <blue>29</blue>
           </color>
          </brush>
         </property>
         <property name="foregroundBrush">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
         <property name="interactive">
          <bool>true</bool>
         </property>
         <property name="sceneRect">
          <rectf>
           <x>0.000000000000000</x>
           <y>0.000000000000000</y>
           <width>0.000000000000000</width>
           <height>0.000000000000000</height>
          </rectf>
         </property>
         <property name="cacheMode">
          <set>QGraphicsView::CacheNone</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_19">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>&lt;---------消息提示---------&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="MessageShow"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
